Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -198,6 +198,9 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 
 		protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
+			if (_disposed)
```

Comment:
I brought up that question a while back on a similar fix. Saw your post on the forum, nice to see that your looking for a better fix :)